file-server/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:latest
+
+# Set the working directory in the container to /app
+WORKDIR /app
+
+# Add the current directory contents into the container at /app
+ADD . /app
+
+# Install any needed packages specified in requirements.txt
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+# Make port 5000 available to the world outside this container
+EXPOSE 5000
```

Comment:
why are we exposing port 5000?

Reply:
Changing this to take from environment var instead

Reply:
Ideally when you deploy to a server, you either use 80 or 443 for http/https reqs